### DISCUSSION TITLE
can not install adonisjs cli on ubuntu 20.4

### DISCUSSION THREAD

**Discussion by shawon122334** [adonisjs/core#3152] [Help] [answered]
I just started learning adonisjs and I am having this error. I dont know what to do.Please help me out thanks
![Screenshot from 2021-09-10 22-55-19](https://user-images.githubusercontent.com/55632206/132900361-fdcc1ae6-3706-49a1-bf83-72fb2235e85e.png)

**Comment by thetutlage** [comment, marked as answer]
The `@adonisjs/cli` is part of the legacy version. Make sure you follow upto date documentation to create a new app. Here's the link https://docs.adonisjs.com/guides/installation

**Comment by shawon122334** [comment]
i could create a new app following the documentation but I can not use adonis command as cli is not installed.by the way thank you

**Reply by RomainLanz** [reply]
You should not use the CLI anymore.

You can use `node ace` instead to have all the generators.
